UPDATING.md

```diff
@@ -165,8 +165,9 @@ Similarly, `DAG.concurrency` has been renamed to `DAG.max_active_tasks`.
 ```python
 dag = DAG(
     dag_id="example_dag",
+    start_date=datetime(2021, 1, 1),
+    catchup=False,
     concurrency=3,
-    start_date=days_ago(2),
```

Comment:
Seeing as this doc is meant to keep users up-to-date on any backwards compat, this was added for completeness but let me know if this is unnecessary.